tests/ffi/disk/test.sh

```diff
@@ -7,12 +7,14 @@
 disk_cleanup
 prepare_test
 
-cat << EOF > /etc/containers/systemd/qm.container.d/oom.conf
+cat << EOF > "${DROP_IN_DIR}"/oom.conf
 [Service]
+OOMScoreAdjust=
```

Comment:
Could you please explain in a comment why we need an extra empty OOMScoreAdjust= ? Just to add a context, in few months I am sure we will forget about this. :(

Reply:
Please refer this tests 
https://github.com/containers/podman/pull/20828/files#diff-320329f5ebb72bf59974a939a83d011595d510cc79f869f1e60eccb80520ab7b

Reply:
Could you please add a comment (short) explaining it?